src/frontend/src/pages/FinancePage/ReimbursementRequestForm/ReimbursementFormView.tsx

```diff
@@ -108,6 +115,18 @@ const ReimbursementRequestFormView: React.FC<ReimbursementRequestFormViewProps>
         handleSubmit(onSubmit)(e);
       }}
     >
+      {!hasSecureSettingsSet && (
+        <Snackbar anchorOrigin={{ vertical: 'top', horizontal: 'center' }} open={true}>
+          <Alert variant="filled" severity="warning">
+            Your secure settings must be set to create a reimbursement request, you can set them
+            <Link component={RouterLink} to={routes.SETTINGS}>
```

Comment:
Let's make this link blue so it has better contrast against the orange. It's particularly hard to see in light mode: 
<img width="657" alt="image" src="https://github.com/Northeastern-Electric-Racing/FinishLine/assets/29521172/d5dedb02-6ea6-4316-837a-565b95542229">